app/src/main/java/org/simple/clinic/patient/PatientSearchResult.kt

```diff
@@ -158,17 +158,17 @@ data class PatientSearchResult(
     fun searchByPhoneNumber(phoneNumber: String, limit: Int): List<PatientSearchResult>
 
     @Query("""
-        SELECT * FROM (
-            SELECT searchResult.*, 0 priority, INSTR(lower(P.fullName), lower(:name)) namePosition FROM 
-            PatientSearchResult searchResult
-            LEFT JOIN Patient P ON P.uuid = searchResult.uuid
-            WHERE P.deletedAt IS NULL AND P.assignedFacilityId = :facilityId AND namePosition > 0
-            UNION
-            SELECT searchResult.*, 1 priority, INSTR(lower(P.fullName), lower(:name)) namePosition FROM 
-            PatientSearchResult searchResult
-            LEFT JOIN Patient P ON P.uuid = searchResult.uuid
-            WHERE P.deletedAt IS NULL AND P.assignedFacilityId != :facilityId AND namePosition > 0
-        )
+        SELECT 
+            searchResult.*, 
+            (
```

Comment:
OMG, the easiest approach. Can't believe I missed it when implementing this feature 😅